docs/CSBR.md

```diff
@@ -251,15 +455,41 @@ In either case, each CA MUST include a link to the official version of these Req
 
 ### 3.2.2 Authentication of organization identity
 
+If the Applicant requests a Certificate that will contain the countryName field and other Subject Identity Information, then the CA SHALL verify the identity of the Applicant, and the authenticity of the Applicant Representative's certificate request using a verification process meeting the requirements of [Section 3.2.2.1](#3221-Authentication-of-organization-identity-for-Non-EV-Code-Signing-Certificates) or [Section 3.2.2.2](#3222-Authentication-of-organization-identity-for-EV-Code-Signing-Certificates) and that is described in the CA's Certificate Policy and/or Certification Practice Statement. The CA SHALL inspect any document relied upon under this Section for alteration or falsification.

```

Comment:
```suggestion
The CA SHALL verify the identity of the Applicant, and the authenticity of the Applicant Representative's certificate request using a verification process meeting the requirements of [Section 3.2.2.1](#3221-Authentication-of-organization-identity-for-Non-EV-Code-Signing-Certificates) or [Section 3.2.2.2](#3222-Authentication-of-organization-identity-for-EV-Code-Signing-Certificates) and that is described in the CA's Certificate Policy and/or Certification Practice Statement. The CA SHALL inspect any document relied upon under this Section for alteration or falsification.

```

Reply:
Fixed in https://github.com/cabforum/code-signing/commit/334ee7b664a9628f0cd4c0481db32f86ff59137b